src/components/breadcrumb/breadcrumb.jsx

```diff
@@ -0,0 +1,102 @@
+import React, { createContext, useContext } from 'react';
+import { cn } from '@/utilities/functions';
+import { ChevronRight, Ellipsis } from 'lucide-react';
+
+const BreadcrumbContext = createContext();
+
+const sizeClasses = {
+	sm: {
+		text: 'text-sm',
+		separator: 'text-sm',
+		separatorIconSize: 16,
+	},
+	md: {
+		text: 'text-base',
+		separator: 'text-base',
+		separatorIconSize: 18,
+	},
+};
+
+export const Breadcrumb = ( { children, size = 'sm' } ) => {
+	const sizes = sizeClasses[ size ] || sizeClasses.sm;
+
+	return (
+		<BreadcrumbContext.Provider value={ { sizes } }>
+			<nav className="flex m-0" aria-label="Breadcrumb">
+				<ul className="m-0 inline-flex items-center space-x-1 md:space-x-1">
+					{ children }
+				</ul>
+			</nav>
+		</BreadcrumbContext.Provider>
+	);
+};
+
+export const BreadcrumbList = ( { children } ) => {
+	return <>{ children }</>;
+};
+
+export const BreadcrumbItem = ( { children } ) => {
+	return <li className="m-0 inline-flex items-center gap-2">{ children }</li>;
+};
+
+export const BreadcrumbLink = ( { href, children, className } ) => {
```

Comment:
can we provide an `as` / `tag` option so that developers can use other components as links? Ex. React Router link.

Reply:
Ok

Reply:
Added